.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: php
 
-php: [5.6, 7.1, hhvm]
```

Comment:
Phpspec 4 is not tested against HHVM and does not support PHP 5.6, but anyway I should have set the min requirements to PHP 7.0, not PHP 7.1.